### ISSUE TITLE
CMI Bungeecord Placeholders doesn't get parsed

### ISSUE THREAD

**Issue by davidgb8246** [Zrips/CMI#7164]
### Description of Issue

The cmi bungeecord module placeholders are sometimes not parsed. I have CMI, CMILib installed on all servers and CMIB and the variables are only parsed on some servers or are parsed incorrectly giving a status of online when that server is actually offline.

Test: I have done a test from the lobby server analyzing that placeholder to see if the survival server is on, receiving a signal that it is online. When trying to parse that placeholder to see if the kitpvp server is on, I get an offline signal when that server is online.

Environment: This is a pterodactyl work environment with the public IP on the bungeecord server (45.xx.xxx.xxx) and the internal IP on the other servers (172.18.0.1).

### Version Information

```shell
--------------------------------------------------
CMI: 9.2.3.5 BungeeCord CMIB  SqLite
CMILib: 1.2.3.3 
Server: Purpur 1.16.5-R0.1-SNAPSHOT
CMI economy: Disabled CMI Chat: Disabled 
Modules -> 55 enabled 1 disabled: votifier
--------------------------------------------------
Bungeecord: Bungeecord-1.8-1.19.x (#1659 release)
Lobby: Purpur 1.16.5 (#1171 release)
KitPvP: Paper 1.8.8 (#445 release)
Survival: Paper 1.19.2 (#123 release)
```

### Errors

```shell
There is no error represented in the console or within the ingame chat.
```

### Relevant Config Sections

```yaml
The plugin is configured by default unless the chat module has been deactivated
```

### Relevant Plugins

PlaceholderAPI-2.11.2

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by davidgb8246** [comment]
Bump issue

**Comment by davidgb8246** [comment]
Bump issue

**Comment by davidgb8246** [comment]
Bump issue

**Comment by Zrips** [comment]
We are talking about %cmi_bungee_onlinestatus_[serverName]% placeholder? And, server which shows up as offline, is it on same host or completely different one?

**Comment by davidgb8246** [comment]
All the servers are in the same host but in different containers. I have pterodactyl installed on the host and all the servers have the IP `127.0.0.1` and the bungeecord server has to public IP `45.xx.xxx.xxx`.

**Comment by davidgb8246** [comment]
> We are talking about %cmi_bungee_onlinestatus_[serverName]% placeholder? And, server which shows up as offline, is it on same host or completely different one?

Yes, the placeholder that has the issue is `%cmi_bungee_onlinestatus_[serverName]%`

**Comment by Zrips** [comment]
I'm adding server ip and port for serverlist command when performing from console for better debuging. This could help out to figure out why your server fails to be recognized as online in case ip and/or port isint correct. Other than this did some testing and placeholder properly updates with server status. There might be slight delay between updates, but it should update its state in a second or two when you send request for this placeholder, you might need to repeat this placeholder request as server status gets updated in async mode, so first request might return old value until it gets updated.

**Comment by davidgb8246** [comment]
The issue is that `%cmi_bungee_onlinestatus_[serverName]%` placeholder gives a result that is not correct. For example, I'm at the KitPvP server and I send a request for this placeholder to the creative server and I get that the creative server is online when never comes online. It's a little bit weird.

**Comment by davidgb8246** [comment]
Bump issue

**Comment by davidgb8246** [comment]
Bump issue

**Comment by davidgb8246** [comment]
Okay, I won't be spamming with bumps in the future. Right now, there is no more information about the issue, so I will have to wait for you to get a resolution to the problem.
